firstpr/fusen.md

```diff
@@ -0,0 +1,5 @@
+I am Fusen Ye from China.
+A beginner of HTML,CSS and JavaScript.
+I want to contribute to the project.
+This my fisrt pr.
```

Comment:
Hi Fusen, congrats on your first PR :tada: 

You just have a small typo in this line. Its `first` actually not `fisrt`.

Thanks!

Reply:
Thank you!
I will be careful next time.

Reply:
Please fix the typo as suggested by Arthur.

Post that we can merge

Reply:
I have fixed the mistake.What should I do?I can not pr agiain.

Reply:
@proful thank you！